docs/source/deployment/k8s.md

```diff
@@ -14,11 +17,103 @@ Alternatively, you can deploy vLLM to Kubernetes using any of the following:
 * [vllm-project/aibrix](https://github.com/vllm-project/aibrix)
 * [vllm-project/production-stack](integrations/production-stack.md)
 
-## Pre-requisite
+## Deployment with CPUs
+
```

Comment:
Please add a warning here saying that CPUs are for demonstration and testing purpose and it's performance will not be on par with GPU.